### PR TITLE
Default for parallelism

### PR DESCRIPTION
**Pull request by liorgreenb**
Added default value for parallelism

based on source code:
https://github.com/webpack/webpack/blob/4972fd8bd6790eb1ee45602c35f38b7f79849b61/lib/Compilation.js#L426

### CODE REVIEW

File: src/content/configuration/other-options.md [hunk 1]
```diff
@@ -100,6 +100,8 @@ Expose custom values into the loader context.
 
 Limit the number of parallel processed modules. Can be used to fine tune performance or to get more reliable profiling results.
 
+default: 100
```
Comment (byzyk) [inline src/content/configuration/other-options.md]: Would you mind moving this to previous paragraph and re-word to `Defaults to '100'.` Thanks!

**Comment by montogeek** [comment]
@liorgreenb Friendly ping

**Comment by liorgreenb** [comment]
Changed to type, as suggested

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**